### PR TITLE
Adds science sunglasses

### PR DESCRIPTION
**Pull request by super12pl**
## About The Pull Request
Adds reskin to beer googles - science sunglasses.
sciencesun on character
sciencesun item

## Why It's Good For The Game
New cosmetic item

## Changelog
:cl:
add: Added science sunglasses
/:cl:

### CODE REVIEW

File: code/modules/clothing/glasses/_glasses.dm [hunk 1]
```diff
@@ -213,6 +213,22 @@
 	. = ..()
 	REMOVE_TRAIT(user, TRAIT_BOOZE_SLIDER, CLOTHING_TRAIT)
 
+/obj/item/clothing/glasses/sunglasses/advanced/reagent_2
+	name = "science sunglasses"
+	desc = "A pair of sunglasses outfitted with apparatus to scan reagents, as well as providing an innate understanding of liquid viscosity while in motion. Has enhanced shielding which blocks flashes."
+	icon_state = "sunhudscience"
+	item_state = "sunhudscience"
+	scan_reagents = TRUE
+
```
Comment (TeomanTheGreat) [inline code/modules/clothing/glasses/_glasses.dm]: I'm not that good at coding but doesn't this need these so they can do the shit the normal sci goggles can do and be acid-proof like them?

	actions_types = list(/datum/action/item_action/toggle_research_scanner)
	resistance_flags = ACID_PROOF
	armor = list("melee" = 0, "bullet" = 0, "laser" = 0, "energy" = 0, "bomb" = 0, "bio" = 0, "rad" = 0, "fire" = 80, "acid" = 100)

Comment (super12pl) [reply code/modules/clothing/glasses/_glasses.dm]: Yeah they do actually need these

**Comment by PowerfulBacon** [comment]
It would probably be better if they were a subclass of the scanner goggles and then would only need flash_protect = 1 rather than all the other stuff

**Comment by super12pl** [comment]
can someone tell me why it didn't pass

**Review by PowerfulBacon — CHANGES_REQUESTED** [review]
use the tab button

File: code/datums/components/crafting/tailoring.dm [hunk 2]
```diff
@@ -114,6 +114,16 @@
...
 	category = CAT_CLOTHING
 
+/datum/crafting_recipe/sunhudscience
+	name = "Science Sunglasses"
+	result = /obj/item/clothing/glasses/science/sciencesun
+	time = 20
+	tools = list(TOOL_SCREWDRIVER, TOOL_WIRECUTTER)
+	reqs = list(/obj/item/clothing/glasses/science = 1,
+				  /obj/item/clothing/glasses/sunglasses = 1,
+				  /obj/item/stack/cable_coil = 5)
```
Comment (PowerfulBacon) [inline code/datums/components/crafting/tailoring.dm:124]: Seems to be 2 spaces at the start of these 2 things here

**Comment by PowerfulBacon** [comment]
Put to vote

File: code/datums/components/crafting/tailoring.dm (hunk 2 above)
Comment (super12pl) [reply code/datums/components/crafting/tailoring.dm:124]: that's how all crafting recipes are done, I just copied beer googles recipe @PowerfulBacon

**Review by PowerfulBacon — APPROVED** [review]

**Comment by MCterra10** [comment]
ok now add engie sunnies

**Pull request merged**